### DISCUSSION TITLE
What's the correct way to call the "Reset show progress" endpoint?

### DISCUSSION THREAD

**Discussion by henrikfroehling** [trakt/trakt-api#329] [Q&A] [answered]
I want to implement this endpoint in my library: https://trakt.docs.apiary.io/#reference/shows/reset-watched-progress/reset-show-progress

The description states that you can optionally specify a "reset_at" date.
But how exactly? As an optional parameter or in the POST body? And in which format?

**Comment by rectifyer** [comment, marked as answer]
POST body, the API docs have an example of the data you can send.

**Reply by henrikfroehling** [reply]
Okay, thanks.
I was not really sure about that, because in the example is only the response body and no request body.
